src/wristpy/core/config.py

```diff
@@ -9,3 +11,22 @@ class Settings(pydantic_settings.BaseSettings):
     LIGHT_THRESHOLD: float = 0.03
     MODERATE_THRESHOLD: float = 0.1
     VIGOROUS_THRESHOLD: float = 0.3
+
+    LOGGING_LEVEL: int = 20
+
+
+def get_logger() -> logging.Logger:
+    """Gets the wristpy logger."""
+    logger = logging.getLogger("wristpy")
+    if logger.handlers:
+        return logger
```

Comment:
This line is never covered. I know when we spoke in the meeting you had to some intricate stuff  with the LogCaptureFixture to get by the if statement, but should we include a test for when the handler already exists?